packages/std/src/math/uint64.rs

```diff
@@ -180,7 +180,7 @@ impl From<Uint64> for u64 {
 
 impl fmt::Display for Uint64 {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "{}", self.0)
+        self.0.fmt(f)
```

Comment:
This change is noteworthy! The difference is now this `Display` implementation handles formatting parameters like `{:018}`. Before it assumed they would always be empty`{}`.

Reply:
This is cool. Can you add to the CHANGELOG?